datacenter/ucp/2.1/guides/admin/manage-users/create-and-manage-teams.md

```diff
@@ -27,6 +27,45 @@ Then choose the list of users that you want to add to the team.
 
 ![](../../images/create-and-manage-teams-3.png){: .with-border}
 
+## Sync team members with your organization's LDAP directory.
+
+If UCP is configured to provision using by syncing with your organization's
```

Comment:
This is really awesome, but if we know that people don't care about reading docs we should show instead of telling. If you notice you're essentially describing the UI.
If you add one or two screenshots we can cut a bit of the text.